src/commands/fetch-compact-filters-node.cpp

```diff
@@ -68,7 +68,14 @@ console_result fetch_compact_filters_node::invoke(std::ostream& output,
     const auto& encoding = get_format_option();
     const hash_digest& stop_hash = get_hash_argument();
     const uint32_t start_height = get_height_argument();
-    const uint8_t type = get_type_argument();
+    const uint16_t filter_type = get_filter_type_argument();
```

Comment:
This should be implemented in a custom config class (parser) to restrict to valid values. Ideally filter types should be named, not numeric.